examples/wdio/jasmine/jasmine.spec.js

```diff
@@ -1,7 +1,7 @@
 describe('webdriver.io page', () => {
     it('should have the right title', () => {
         browser.url('https://webdriver.io')
-        const title = browser.getTitle()
-        expect(title).toBe('WebdriverIO · Next-gen WebDriver test framework for Node.js')
+        expect(browser.getTitle())
```

Comment:
We can use here expect(browser).toHaveTitle(...)

Reply:
@mgrybyk it fails with `TypeError: expect(...).toHaveTitle is not a function`, when running:

```
describe('webdriver.io page', () => {
    it('should have the right title', () => {
        browser.url('https://webdriver.io')
        expect(browser).toHaveTitle('WebdriverIO · Next-gen WebDriver test framework for Node.js')
    })
})
```

Reply:
I'll take a look